backend/src/Request/Admin/StoreOwnerPayment/AdminStoreOwnerPaymentCreateRequest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Request\Admin\StoreOwnerPayment;
+
+use App\Entity\StoreOwnerProfileEntity;
+
+class AdminStoreOwnerPaymentCreateRequest
+{
+    private float $amount;
+
+    private int|StoreOwnerProfileEntity $store;
+
+    /**
+     * Get the value of store
+     */ 
+    public function getStore()
+    {
+        return $this->store;
+    }
+
+    /**
+     * Set the value of store
+     *
+     * @return  self
+     */ 
+    public function setStore(StoreOwnerProfileEntity $store)
```

Comment:
While you set that the **store** variable could be of type _int_, why not declare that as a type hint besides the _StoreOwnerProfileEntity_?

Reply:
the variable could be of type int because the front is sent as a int , but in function must can of storeProfileEntity.
I send to you  explain on whatsapp